app/views/spree/taxons/_pagination.html.erb

```diff
@@ -14,6 +14,6 @@
 </div> -->
 <% if @products.respond_to?(:total_pages) %>
     <div class="pagination-wrap">
-        <%= paginate @products, window: 2, theme: 'twitter-bootstrap-4' %>
+        <%= paginate @products, window: 2, theme: 'twitter-bootstrap-4', total_product_count: @taxon.products.length  %>
```

Comment:
change the variable name from total_product_count to total_products_count. As the count is for multiple products